docs/reference/transform/examples.asciidoc

```diff
@@ -275,58 +274,83 @@ POST _transform/_preview
 ----------------------------------
 // TEST[skip:setup kibana sample data]
 
-<1> This range query limits the {transform} to documents that are within the 
-last 30 days at the point in time the {transform} checkpoint is processed. For 
-batch {transforms} this occurs once.
-<2> This is the destination index for the {dataframe}. It is ignored by 
-`_preview`.
-<3> The data is grouped by the `clientip` field. 
-<4> This `scripted_metric` performs a distributed operation on the web log data 
+<1> This is the destination index for the {transform}.
+<2> This configures the {transform} in continuous mode using the field `timestamp`
```

Comment:
Not mandatory, but I don't see that we talk about "continous mode" in the docs so this might be clearer:
```suggestion
<2> Configures the {transform} to run continuously. It uses the `timestamp` field
```